docs/language.md

```diff
@@ -6001,6 +6195,21 @@ let counterRef = countCap.borrow()
 let counterRef2 = publicAccount.borrow<&Counter>(/storage/counter)
 ```
 
+### Account Creation
+
+Accounts can be created by calling the `AuthAccount` constructor
+and passing the account that should pay for the account creation for the `payer` parameter.
+
+For example, to create an account and have the signer of the transaction pay for the account creation:
+
+```cadence,file=account-creation.cdc
+transaction {
+    prepare(signer: AuthAccount) {
+        let account = AuthAccount(payer: signer)
```

Comment:
Should we include adding keys to an account so it shows the usual way of creating an AuthAccount? or do we want to keep it simple?

Reply:
Yep, adding that, good point

Reply:
Documented that in f3dbda6